launch/s3000_laser.launch

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
 <launch>
-  <param name="port" value="/dev/ttyUSB1" type="string"/>
+  <param name="~port" value="/dev/ttyUSB1" type="string"/>
```

Comment:
Also, for "example" launchfiles shipped in sensor packages, please arg-ify parameters like the port.

cf. https://github.com/ros-drivers/flir_ptu/blob/master/flir_ptu_driver/launch/ptu.launch#L2